fictadvisor-api/src/v2/api/services/ScheduleService.ts

```diff
@@ -134,9 +134,11 @@ export class ScheduleService {
     week = week || await this.dateService.getCurrentWeek();
     day = day || (await this.dateService.getCurrentDay()).day;
 
+    const query = undefined;
+
     const { startOfDay } = await this.dateService.getSpecificDayInWeek(week, day);
 
-    const events  = await this.getGroupEventsForTelegram(groupId, week, userId);
+    const events  = await this.getGroupEventsForTelegram(groupId, week, userId, query);
```

Comment:
You can make `query` param nullable in `getGroupEventsForTelegram` method and do not pass extra empty param